CONTRIBUTING.md

```diff
@@ -1,120 +1,104 @@
-# Contributing Guidelines
+# Contributing to Textual
 
-🎉 **First of all, thanks for taking the time to contribute!** 🎉
+First of all, thanks for taking the time to contribute to Textual!
 
-## 🤔 How can I contribute?
+## How can I contribute?
 
-**1.** Fix issue
+You can contribute to Textual in many ways:
 
-**2.** Report bug
+ 1. [Report a bug](https://github.com/textualize/textual/issues/new?title=%5BBUG%5D%20short%20bug%20description&template=bug_report.md)
+ 2. Propose a new feature
+ 3. Work on a previously opened issue
+ 4. Improve the documentation
+ 5. Talk/write about Textual online
 
-**3.** Improve Documentation
 
+## Setup
 
-## Setup 🚀
-You need to set up Textualize to make your contribution. Textual requires Python 3.7 or later (if you have a choice, pick the most recent Python). Textual runs on Linux, macOS, Windows, and probably any OS where Python also runs.
+To make a code or documentation contribution you will need to set up Textual locally.
+You can follow these steps:
 
-### Installation
+ 1. Make sure you have Poetry installed ([see instructions here](https://python-poetry.org))
+ 2. Clone the Textual repository
+ 3. Run `poetry shell` to create a virtual environment for the dependencies
+ 4. Run `poetry install` to install all dependencies
+ 5. Make sure the latest version of Textual was installed by running the command `textual --version`
+ 6. Install the pre-commit hooks with the command `pre-commit install`
 
-**Install Texualize via pip:**
-```bash
-pip install textual
-```
-**Install [Poetry](https://python-poetry.org/)**
-```bash
-curl -sSL https://install.python-poetry.org | python3 -
-```
-**To install all dependencies, run:**
-```bash
-poetry install --all 
-```
-**Make sure everything works fine:**
-```bash
-textual --version
-```
-### Demo
+## Demo
 
-Once you have Textual installed, run the following to get an impression of what it can do:
+Once you have Textual installed, run the Textual demo to get an impression of what Textual can do and to double check that everything was installed correctly:
 
 ```bash
 python -m textual
 ```
-If Texualize is installed, you should see this:
-<img width="848" alt="demo" src="https://github.com/clairecharles/textual/assets/67120042/62fd53c0-7ad6-4429-8751-5a713180b836">
 
-## Make contribution
-**1.** Fork [this](repo) repository.
+## Guidelines
 
-**2.** Clone the forked repository.
+- Make sure to read the issue instructions carefully. If something isn't clear, ask for clarification!
```

Comment:
Hope you don't mind me sticking my nose in, but I'm not sure assume "issue instructions" makes sense here?

Reply:
_I_ am not sure I understand your question. The “issue instructions” are the instructions that can be found on the issue that the contributor is addressing.
How would you word it?

Reply:
Sorry, what I meant was that most issues probably don't have _instructions_ exactly, unless a specific task or feature request. I think this bullet is from the EuroPython sprint where this made sense, but perhaps not for issues more generally?

It was only a minor nit, I think these guidelines are much clearer now!